articles/quickstart/_includes/_auth0-react-classes-info.md

```diff
@@ -0,0 +1,5 @@
+<!-- markdownlint-disable MD041 -->
+
+:::note
+This guide focuses on using the `useAuth0()` custom React Hook. If you are using class components, check out [these samples using the `withAuth0()` higher-order component](https://github.com/auth0/auth0-react#use-with-a-class-component).
```

Comment:
What is the reason for the react stuff to be includes in this PR? Is this expected ? (there are more files like this)

Reply:
The branch is based on #9763, which doesn't make review all that easy. Try to ignore the React stuff if possible, but continue to raise if you're not sure.

Reply:
The diff will shorten once that PR is merged.